docs/container.md

```diff
@@ -0,0 +1,69 @@
+<!--
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+
+# Containerized imageio server
+
+## Overview
+
+ovirt-imageio provides an image for running the ovirt-imageio-daemon
+in a container.
+
+Using a container, users can run an image transfer pod for every
+transfer, exposing the pod for remote access, and connecting the
+relevant storage the pod. When the transfer is done, the user will
+unexpose the pod and terminate it.
+
+In this mode the imageio server will serve one ticket, injected into the
+pod when starting the pod, instead of adding a ticket using the control
+service. The control service is actually disabled.
+
+To create the container run:
+
+    make container
+
+This creates localhost/ovirt-imageio container image using podman
+configured for [rootless environment](https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md).
+
+
+To run the container we need to load a ticket and create the image in the
+mounted images folder. Note that the path for the file in the ticket needs
+to match the path where we mounted the images folder in the container.
+
+Alternatively, we can mount the nbd socket path in the container
+and request the image with a nbd ticket.
+
+In order to specify the path to the ticket that we want to load,
+you can use an environment variable, namely `TICKET_PATH`, that will
+allow you to define the full path for the volume mount point
+and the ticket name.
```

Comment:
This text is missing the critical part - mounting an image in the container.

The text should be something like:

1. mounting image in the container
2. exposing the image via a ticket

The text should show the contents of a ticket, since users must create a valid
ticket to use the container.